app/models/member.rb

```diff
@@ -1,13 +1,19 @@
 require 'hashie'
 
-class Member
-
+class Member < Hashie::Mash
   def self.list
-    MEMBER_CONFIG.map { |a|
-      Hashie::Mash.new(:name =>        a['name'],
-                       :github_name => a['github_name'],
-                       :twitter =>     a['twitter'],
-                       :avatar_file => a['avatar_file'])
-    }
+    MEMBER_CONFIG.map(&method(:new))
+  end
+
+  def avatar_file_path
+    "member/#{avatar_file}" if avatar_file.present?
```

Comment:
typo here that breaks all the existing file based avatars should be `"members/#{avatar_file}" if avatar_file.present?` **members**